### PR TITLE
Add local_todo integration documentation

### PR DESCRIPTION
**Pull request by allenporter**
## Proposed change

Add local_todo integration documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102627
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4812
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by allenporter** [comment]
I could use some assistance understanding the check failure:
```
8:43:48 PM: build-image version: 0c5695ad82eb9b0111196b4ab87e76fc88dfb543 (focal)
8:43:48 PM: buildbot version: 44bdaa9bbaa9912838fe50ed0826ccd09db9f453
8:43:49 PM: Fetching cached dependencies
8:43:49 PM: Starting to download cache of 1010.4MB
8:44:02 PM: Finished downloading cache in 13.69s
8:44:02 PM: Starting to extract cache
8:44:06 PM: Failed during stage "preparing repo": git ref pull/29496/head does not exist
8:44:06 PM: Finished extracting cache in 3.766s
8:44:06 PM: Finished fetching cache in 17.558s
8:44:06 PM: Starting to prepare the repo for build
8:44:06 PM: git ref pull/29496/head does not exist or you do not have permission
8:44:06 PM: Failing build: Failed to prepare repo
```

File: source/_integrations/local_todo.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Local To-do List
```
Comment (c0ffeeca7) [inline source/_integrations/local_todo.markdown]:
```suggestion
title: Local to-do list
```

File: source/_integrations/local_todo.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+description: Instructions on how to use Local To-do Lists in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/local_todo.markdown]:
```suggestion
description: Instructions on how to use local to-do lists in Home Assistant.
```

File: source/_integrations/local_todo.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ha_config_flow: true
+ha_domain: local_todo
+ha_platforms:
+  - todo
+ha_codeowners:
+  - '@allenporter'
+ha_integration_type: integration
+---
+
+The Local To-do List integration allows you to create a To-do Lists in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/local_todo.markdown]:
```suggestion
The **Local to-do list** integration allows you to create to-do lists in Home Assistant.
```

File: source/_integrations/local_todo.markdown [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+To-do Lists are shown on the To-do List dashboard for tracking items and whether
+or not they have been completed.
+
+See the [To-do List integration](/integrations/todo) for additional details
```
Comment (c0ffeeca7) [inline source/_integrations/local_todo.markdown]:
```suggestion
See the [**To-do list** integration](/integrations/todo) for additional details
```

File: source/_integrations/local_todo.markdown [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+about To-do List entities.
```
Comment (c0ffeeca7) [inline source/_integrations/local_todo.markdown]:
```suggestion
about to-do list entities.
```

File: source/_integrations/local_todo.markdown [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+ha_domain: local_todo
+ha_platforms:
+  - todo
+ha_codeowners:
+  - '@allenporter'
+ha_integration_type: integration
+---
+
+The **Local to-do list** integration allows you to create to-do lists in Home Assistant.
+To-do Lists are shown on the To-do List dashboard for tracking items and whether
```
Comment (c0ffeeca7) [inline source/_integrations/local_todo.markdown]:
```suggestion
To-do lists are shown on the **To-do list** dashboard for tracking items and whether
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @allenporter :+1:

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**